src/Multiselect.vue

```diff
@@ -1,166 +1,222 @@
 <template>
   <div
     :tabindex="searchable ? -1 : tabindex"
-    :class="{ 'multiselect--active': isOpen, 'multiselect--disabled': disabled, 'multiselect--above': isAbove }"
-    @focus="activate()"
+    :class="{
+      'multiselect--active': isOpen,
+      'multiselect--disabled': disabled,
+      'multiselect--above': isAbove,
+    }"
     @blur="searchable ? false : deactivate()"
     @keydown.self.down.prevent="pointerForward()"
     @keydown.self.up.prevent="pointerBackward()"
     @keypress.enter.tab.stop.self="addPointerElement($event)"
     @keyup.esc="deactivate()"
     class="multiselect"
-    role="combobox"
-    :aria-owns="'listbox-'+id">
-      <slot name="caret" :toggle="toggle">
-        <div @mousedown.prevent.stop="toggle()" class="multiselect__select"></div>
+  >
+    <!-- put this in the upper div @focus="activate()" -->
+    <slot name="caret" :toggle="toggle">
+      <div @mousedown.prevent.stop="toggle()" class="multiselect__select"></div>
+    </slot>
+    <slot name="clear" :search="search"></slot>
+    <div ref="tags" class="multiselect__tags">
+      <slot
+        name="selection"
+        :search="search"
+        :remove="removeElement"
+        :values="selectedOptions"
+        :is-open="isOpen"
+      >
+        <div class="multiselect__tags-wrap" v-show="visibleValues.length > 0">
+          <draggable
+            class="list-group"
+            tag="ul"
+            v-model="selectedOptions"
```

Comment:
I'm not too sure about this, but maybe `selectedOptions` wouldn't be needed if the `v-model` was split to `v-bind:value` and `v-on:input`. Relevant docs: https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components
```suggestion
            v-bind:value="value"
            v-on:input="(v) => $emit('input', v)"
```